examples/step-68/doc/intro.dox

```diff
@@ -0,0 +1,234 @@
+associated with each cell and its particles. Repartitioning the mesh then
+accounts for this combined computational load instead of the simplified
+assumption of the number of cells.
+
+In this section we only discussed the particle-specific challenges in distributed
+computation. Parallel challenges that particles share with
+finite-element solutions (parallel output, data transfer during mesh
+refinement) can be addressed with the solutions found for
+finite-element problems already discussed in other examples.
+
+<h3>The testcase</h3>
+
+In the present step, we use particles as massless tracers to illustrate
+the dynamics of a particular vortical flow : the Rayleigh-Kothe Vortex. This flow pattern
+is generally used as a complex test case for interface tracking methods
+(e.g. volume-of-fluid and level set approches) since
+it leads to strong rotation and elongation of the fluid (Blais et al. 2013).
+
+The stream function $\Psi$ of this Rayleigh-Kothe vortex is defined as:
+
+@f[
+\Psi = \frac{1}{\pi} sin^2 (\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f]
+where $T$ is the period of the flow. The velocity profile in 2D ($\textbf{u}=[u,v]^T$) is :
+@f{eqnarray*}
+   u &=&  - \frac{\partial\Psi}{\partial y} = -2 \sin^2 (\pi x) \sin (\pi y) \cos (\pi y)  \cos \left( \pi \frac{t}{T} \right)\\
+   v &=&  \frac{\partial\Psi}{\partial x} = 2 \cos(\pi x) \sin(\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f}
+
+It can be seen that this velocity reverses periodically due to the term
+$\cos \left( \pi \frac{t}{T} \right)$ and that material will end up at its
+starting position after every period of length $t=2T$. We will run this tutorial
+program for exactly one period and compare the final particle location to the
+initial location to illustrate this flow property. This example uses the testcase
+to produce two models that handle the particles
+slightly differently. The first model prescribes the exact analytical velocity
+solution as the velocity for each particle. Therefore in this model there is no
+error in the assigned velocity to the particles, and any deviation of particle
+positions from the analytical position at a given time results from the error
+in solving the equation of motion for the particle. In the second model the
+analytical velocity field is first interpolated to a finite-element vector
+space (to simulate the case that the velocity was obtained from solving a
+finite-element problem). This finite-element "solution" is then evaluated at
+the locations of the particles to solve their equation of motion. The
+difference between the two cases allows to assess whether the chosen
+finite-element space is sufficiently accurate to advect the particles with the
+optimal convergence rate of the chosen particle advection scheme, a question
+that is important in practice to determine the accuracy of the combined
+algorithm (see e.g. Gassmöller et al., 2019).
+
+
+
+<h3>References</h3>
```

Comment:
If you replace all references above by `@cite`, you don't need this section any more.